REFERENCE.md

```diff
@@ -0,0 +1,133 @@
+# Reference
```

Comment:
@bsper2 How is this file generated? I haven't been able to figure this out yet.

Reply:
The command itself is listed in the REFERENCE.md row on this page : https://wiki.ncsa.illinois.edu/pages/viewpage.action?pageId=139134991

There is also this page on the wiki that has some other reference links included:
https://wiki.ncsa.illinois.edu/display/PUPPET/PDK+-+Puppet+Development+Kit#PDKPuppetDevelopmentKit-YamlLintCheck